cookbook/form/data_transformers.rst

```diff
@@ -63,31 +171,36 @@ for converting to and from the issue number and the ``Issue`` object::
                 return '';
```

Comment:
I think it's ok - it's the string that'll be populated in the box, so it *is* meant to be a string. If anything, we would want to cast `$issue->getId()` to string below, but I think that's overkill